README.md

```diff
@@ -1 +1,153 @@
-# Hello-world
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+## HTML+CSS Setup
+
+**HTML+CSS Setup** is an open source project aimed at building a safer, secure, fast, efficient and more realiable website
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Key Features
+
+- npm packages
+- html, renders the project
+
```

Comment:
this section is to describe the features built in the project, in this case it displays a basic hello world styled message.